forms/src/main/java/swim/basic/BarType.java

```diff
@@ -0,0 +1,92 @@
+package swim.basic;
+
+import swim.recon.Recon;
+import swim.structure.*;
```

Comment:
For whatever reason, we typically don't use `.*` imports at swim. IntelliJ should let you alt+enter to expand them automatically. Alt+enter is just great in general.